docs/mpcie_modules.md

```diff
@@ -207,6 +207,29 @@ can be done, for example in Debian OS, by performing those steps:
     sudo apt-get install firmware-realtek
     ```
 
+## SATA controllers
+
+### ASMedia ASM1061
+
+ASM1061 was tested with Delock 95233 mPCIe 2xSATA module.
```

Comment:
Add single sentence that module is not supported. Capital letters would be good @piotr-kleins

Reply:
Done